app/javascript/stimulus/__mocks__/mutationObserver.js

```diff
@@ -0,0 +1,23 @@
+// Sourced from https://github.com/stimulusjs/stimulus/issues/34
+// and https://shime.sh/testing-stimulus
+
+// Setup MutationObserver shim since jsdom doesn't
+// support it out of the box.
```

Comment:
I'm fairly new to Jest, so I'm not sure if this is the "right" way to use a mock. I tried using this as a setup script, like the article I linked suggested, but that wasn't working.

Open to suggestions on a better way to do this.